configfs.c

```diff
@@ -247,6 +247,19 @@ int tcmu_set_cfgfs_ul(const char *path, unsigned long val)
 	return tcmu_set_cfgfs_str(path, buf, strlen(buf) + 1);
 }
 
+bool tcmu_is_ring_reset_support(struct tcmu_device *dev)
```

Comment:
Let's make this generic something like

tcmu_cfgfs_file_exists or tcmu_cfgfs_file_is_supported where one of the args is the file or dir or path to check.

Reply:
Yes, agree and will fix it